<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadingForm</class>
 <widget class="QWidget" name="UploadingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Syncing Process</string>
  </property>
  <widget class="QLabel" name="app_logo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>531</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
    </font>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_detail">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
    </font>
   </property>
   <property name="text">
    <string>Show details</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pt_detail">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>531</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_background">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
    </font>
   </property>
   <property name="text">
    <string>Run in background</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_step">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
    </font>
   </property>
   <property name="text">
    <string>Step 1/4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_cancel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_pause">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
    </font>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>We're syncing your data with our plateform. It may take some times to do that. 
Please wait for a while</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
